prog/vm/host_nexus.rb

```diff
@@ -206,11 +206,25 @@ def before_run
       hop_prep_reboot
     end
 
+    when_checkup_set? do
+      hop_unavailable if !available?
+    end
+
     Clog.emit("vm host utilization") { {vm_host_utilization: vm_host.values.slice(:location, :arch, :total_cores, :used_cores, :total_hugepages_1g, :used_hugepages_1g, :total_storage_gib, :available_storage_gib).merge({vms_count: vm_host.vms_dataset.count})} }
 
     nap 30
   end
 
+  label def unavailable
+    Prog::PageNexus.assemble("#{vm_host} is unavailable", vm_host.ubid, "VmHostUnavailable", vm_host.ubid)
```

Comment:
I designed it poorly. `related_resources` should be an array.

Reply:
I made a small chance to PageNexus.assemble. How does it look like now?